index.html

```diff
@@ -69,5 +69,184 @@ <h2 class="connect">Let's connect</h2>
         </ul>
       </div>
     </section>
+
+    <section class="projects">
```

Comment:
- [ ] Great job in implementing the task, Please ensure that you adjust the gaps between your cards in the layout.

The table below is the difference between yours and the given design. 

<table>
    <tr>
      <td>Your design</td>
      <td>Figma Design/td>
    </tr>
    <tr>
      <td>

![Gaps in bewteen](https://github.com/DHE19/portfolio-microverse/assets/43528594/e135d0db-a44a-4275-961d-f21efa08fc61)

</td>
      <td>

![Gap from figma](https://github.com/DHE19/portfolio-microverse/assets/43528594/e929fed6-77c0-4815-bbd4-c6d95bbfd8ad)

</td>
    </tr>
  </table>